strawberryfields/gbs/sample.py

```diff
@@ -294,28 +284,76 @@ def to_subgraphs(samples: list, graph: nx.Graph) -> list:
+    gates are applied to the first N modes:
 
     #. Interferometer ``U1``
-    #. Squeezing on all modes with parameters ``r``
+    #. Squeezing on all N modes with parameters ``r``
     #. Interferometer ``U2``
-    #. Displacement on all modes with parameters ``alpha``
+    #. Displacement on all N modes with parameters ``alpha``
+
+    In the last step, the number of photons in each of the 2N modes is measured.
+
+    In the zero temperature case, only N modes are considered. The four gates mentioned above are
+    applied to N vacuum states and then the number of photons in each mode is measured. This
+    makes the zero temperature sampling significantly faster than the finite temperature one.
+    However, to keep the sizes of the samples consistent and temperature independent, we add N
+    zeros to the end of each zero temperature sample at the very end of this function.
+
+    **Example usage:**
 
-    Note that, since the gates are applied to the vacuum, the first interferometer has no effect.
-    It is nevertheless included so that the inputs to this function follow a standard decomposition
-    of Gaussian unitaries shown :ref:`above <decomposition>`.
+>>> t = np.array([0., 0., 0., 0., 0., 0., 0.])
+
+>>> U1 = np.array(
+>>>     [[-0.07985219, 0.66041032, -0.19389188, 0.01340832, 0.70312675, -0.1208423, -0.10352726],
+>>>     [0.19216669, -0.12470466, -0.81320519, 0.52045174, -0.1066017, -0.06300751, -0.00376173],
+>>>     [0.60838109, 0.0835063, -0.14958816, -0.34291399, 0.06239828, 0.68753918, -0.07955415],
+>>>     [0.63690134, -0.03047939, 0.46585565, 0.50545897, 0.21194805, -0.20422433, 0.18516987],
+>>>     [0.34556293, 0.22562207, -0.1999159, -0.50280235, -0.25510781, -0.55793978, 0.40065893],
+>>>     [-0.03377431, -0.66280536, -0.14740447, -0.25725325, 0.6145946, -0.07128058, 0.29804963],
+>>>     [-0.24570365, 0.22402764, 0.003273, 0.19204683, -0.05125235, 0.3881131, 0.83623564],
+>>>     ]
+>>> )
+
+>>> r = np.array(
+>>>     [0.09721339, 0.07017918, 0.02083469, -0.05974357, -0.07487845, -0.1119975, -0.1866708]
+>>> )
+
+>>> U2 = np.array(
+>>>     [
+>>>     [-0.07012006, 0.14489772, 0.17593463, 0.02431155, -0.63151781, 0.61230046, 0.41087368],
+>>>     [0.5618538, -0.09931968, 0.04562272, 0.02158822, 0.35700706, 0.6614837, -0.326946],
+>>>     [-0.16560687, -0.7608465, -0.25644606, -0.54317241, -0.12822903, 0.12809274, -0.00597384],
+>>>     [0.01788782, 0.60430409, -0.19831443, -0.73270964, -0.06393682, 0.03376894, -0.23038293],
+>>>     [0.78640978, -0.11133936, 0.03160537, -0.09188782, -0.43483738, -0.4018141, 0.09582698],
+>>>     [-0.13664887, -0.11196486, 0.86353995, -0.19608061, -0.12313513, -0.08639263, -0.40251231],
+>>>     [-0.12060103, -0.01169781, -0.33937036, 0.34662981, -0.49895371, 0.03257453, -0.70709135],
+>>>     ]
+>>> )
+
+>>> alpha = np.array(
+>>>     [0.15938187, 0.10387399, 1.10301587, -0.26756921, 0.32194572, -0.24317402, 0.0436992]
+>>> )
```

Comment:
Anything we can do to avoid this lonely parenthesis occupying a whole line?

Reply:
I will bring them to previous line. I think black does some of such corrections.